src/apps/opening-hours-editor/DialogFomular.tsx

```diff
@@ -0,0 +1,70 @@
+// I dont know why eslint is complaining about label-has-associated-control
+// as the label is associated with the input field. I will disable it for now.
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from "react";
+import { EventImpl } from "@fullcalendar/core/internal";
+import { extractTime, updateEventTime } from "./helper";
+
+type DialogFomularProps = {
+  eventInfo: EventImpl;
+  handleEventEditing: (eventInfo: EventImpl) => void;
+};
+
+const DialogFomular: React.FC<DialogFomularProps> = ({
+  eventInfo,
+  handleEventEditing
+}) => {
+  const [title, setTitle] = useState(eventInfo.title);
+  const [startTime, setStartTime] = useState(extractTime(eventInfo.startStr));
+  const [endTime, setEndTime] = useState(extractTime(eventInfo.endStr));
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    eventInfo.setProp("title", title);
+    const startDate = updateEventTime(eventInfo.startStr, startTime);
+    const endDate = updateEventTime(eventInfo.endStr, endTime);
+    eventInfo.setDates(startDate, endDate);
+
```

Comment:
I think we should stretch us as far as possible to pass date objects around. Or dayjs objects since that has been the chosen library so far (I dont care which we choose to use as long as we pass objects around).
So as soon as possible we convert date/time to objects. So we are never in doubt what we are dealing with and we can make use of a more strict typing, instead of just strings.